modules/apps/portal-remote/portal-remote-cors-impl/src/main/java/com/liferay/portal/remote/cors/internal/url/pattern/mapper/StaticSizeTrieURLPatternMapper.java

```diff
@@ -120,10 +120,6 @@ else if ((urlPath.length() > 1) &&
 			}
 		}
 
-		if (row > (_maxURLPatternLength - 1)) {
-			currentBitmask = 0;
```

Comment:
may have something to do with pattern like **/documents/user1/folder1/**, let me test.